apps/minifront/src/components/ibc/ibc-in/hooks.ts

```diff
@@ -102,10 +107,63 @@ interface UseCosmosChainBalancesRes {
   error: unknown;
 }
 
+const getIconFromAsset = (asset: Asset): string | undefined => {
+  const logoUri = asset.logo_URIs?.svg || asset.logo_URIs?.png;
+  if (logoUri) {
+    return logoUri;
+  }
+
+  if (asset.images?.length) {
+    const first = asset.images[0]!;
+    return first.svg || first.svg;
+  }
+  return undefined;
+};
```

Comment:
There are two places where icons can live. On osmosis, penumbra does not have `logo_URIs` so going to pull from both fields to see if an image is available.